### PR TITLE
tweak 1.22 blog post to desktop realities

### PR DESCRIPTION
**Pull request by r10s**
that should be merged when 1.22 desktop is actually out.

### CODE REVIEW

File: _posts/2021-08-24-updates.md [hunk 1]
```diff
@@ -70,7 +71,8 @@ The third big part: We made the **connectivity visible:**
 
 <img src="../assets/blog/2021-08-connectivity.png" style="width:260px; clear:both; margin-bottom:.2em;" alt="Screenshot showing 'Not connected' in the title bar" />
 
-The **title bar** now shows if you are not connected for whatever reason,
+The **title bar** (Android/iOS) or a **little banner** (Desktop)
```
Comment (Simon-Laux) [inline _posts/2021-08-24-updates.md]: I would call it toast, not banner. banner needs to be something at the top in my mind.

Comment (r10s) [reply _posts/2021-08-24-updates.md]: hm, the term "toast" is not widely known outside development circles. maybe say "pop-up"?

Comment (Simon-Laux) [reply _posts/2021-08-24-updates.md]: popup is even worse, the word is normally associated with ad/scam popup windows i think

File: _posts/2021-08-24-updates.md [hunk 2]
```diff
@@ -84,8 +86,7 @@ also there - stay tuned :)
```
Comment (r10s) [inline _posts/2021-08-24-updates.md:84]: we could also strike the sentence "The connectivity view will get some..." and replace it by an image of connectivity view with quota shown as a percentage-bar - this is what i had in mind when writing that sentence.

Comment (Simon-Laux) [reply _posts/2021-08-24-updates.md:84]: sounds good

File: _posts/2021-08-24-updates.md (hunk 1 above)
Comment (r10s) [reply _posts/2021-08-24-updates.md]: or say "little notification" or just "notification", wikipedia files all that under "pop-up notification" https://en.wikipedia.org/wiki/Pop-up_notification

File: _posts/2021-08-24-updates.md (hunk 2 above)
Comment (r10s) [reply _posts/2021-08-24-updates.md:84]: did so.

File: _posts/2021-08-24-updates.md (hunk 1 above)
Comment (r10s) [reply _posts/2021-08-24-updates.md]: i changed that to "toast notification" :)

**Comment by r10s** [comment]
> that should be merged when 1.22 desktop is actually out.

it is available in apple's app store, so merging that to not let device message point to outdated information.

**Pull request merged**